src/lib/registration/ui/migration_repos_workflow.rb

```diff
@@ -401,7 +402,8 @@ def migration_confirmed?(base_product, activations)
         # TRANSLATORS: Continue/Cancel popup question, confirm the migration by user,
         # %s is a product name, e.g. "SUSE Linux Enterprise Server 15"
         message = _("The product %s\nis already activated on this system.\n\n" \
-                    "Migrating again to the same product might not work properly.") % base_product.label
+                    "Migrating again to the same product might not work properly.") %
```

Comment:
Huh, why did we get away with `%` instead of `format` here?

Reply:
I see no change. Still "%" in diff